frontend/src/services/user.ts

```diff
@@ -0,0 +1,22 @@
+import {AuthCredentials} from "../models/authCredentials";
+import {ILoggedUser} from "../stores/redux/loggedUser/loggedUser";
+import {UserEntity} from "../models/userEntity";
+import {UserType} from "../types/UserType";
+
+export const logInService = async (authCredentials: AuthCredentials): Promise<ILoggedUser> => {
+    const token = "eyJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJjdXN0b21lcjFAZ21haWwuY29tIiwiaWF0IjoxNzExOTYyMDM4fQ.g5oDNUFe6MwsfP7syojOKMTd75OXiSL0VGCPEc2aoHg";
+    const userEntity: UserEntity = {
+        id: 0,
+        email: authCredentials.email,
+        name: "User",
+        createdAt: new Date(),
+        updatedAt: new Date(),
+    }
+    return {
+        id: userEntity.id,
+        email: userEntity.email,
+        username: userEntity.name,
+        userType: UserType.Customer,
+        token: token
+    };
```

Comment:
It's not a task of log in service. You have to create separate service to get a user